source/PlanetPanel.cpp

```diff
@@ -45,9 +45,9 @@ using namespace std;
 
 
 
-PlanetPanel::PlanetPanel(PlayerInfo &player, function<void()> callback)
+PlanetPanel::PlanetPanel(PlayerInfo &player, function<void()> callback, const Planet &planet)
 	: player(player), callback(callback),
-	planet(*player.GetPlanet()), system(*player.GetSystem()),
+	planet(planet), system(*player.GetSystem()),
```

Comment:
Should the system be the player's system even though the planet might not be?

Reply:
fixed with next commit